code/game/objects/items/devices/button.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/money_button
+	name = "red button"
+	desc = "If you press this button, you will get 1000 credits. HOWEVER, one random crewmember will die, which can be you. If you press the button, the button will transfer to someone else."
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "bigred"
+	item_state = "electronic"
+	w_class = WEIGHT_CLASS_TINY
+	origin_tech = "syndicate=2"
+	var/mob/living/last_pressed_person
+
+/obj/item/money_button/attack_self(mob/user)
+	if(!istype(user))
+		return
+
+	if(prob(10) && last_pressed_person) // Fuck you for selling your colleague's lives for a measly 1k credits.
+		last_pressed_person.death()
+	else
+		var/list/copy_list = GLOB.alive_mob_list
```

Comment:
shouldn't this call .Copy()?

Reply:
It should, will fix this when I get home